doc/sphinx/changes.rst

```diff
@@ -765,6 +765,51 @@ A few bug fixes and documentation improvements, in particular:
   fixes `#4741 <https://github.com/coq/coq/issues/4741>`_,
   by Helge Bahmann).
 
+Changes in 8.10.2
+~~~~~~~~~~~~~~~~~
+
+**Kernel**
+
+- Fixed a critical bug of template polymorphism and nonlinear universes
+  (`#11128 <https://github.com/coq/coq/pull/11128>`_,
+  fixes 8.6-regression `#11039 <https://github.com/coq/coq/issues/11039>`_,
+  by Gaëtan Gilbert).
+
+- Fixed an anomaly “Uncaught exception Constr.DestKO” on :g:`Inductive`
+  (`#11052 <https://github.com/coq/coq/pull/11052>`_,
+  fixes 8.10-regression `#11048 <https://github.com/coq/coq/issues/11048>`_,
+  by Gaëtan Gilbert).
+
+- Fixed an anomaly “not enough abstractions in fix body”
+  (`#11014 <https://github.com/coq/coq/pull/11014>`_,
+  fixes `#8459 <https://github.com/coq/coq/issues/8459>`_,
+  by Gaëtan Gilbert).
+
+**Notations**
+
+- Fixed an 8.10 regression related to the printing of coercions associated to notations
+  (`#11090 <https://github.com/coq/coq/pull/11090>`_,
+  fixes `#11033 <https://github.com/coq/coq/issues/11033>`_, by Hugo Herbelin).
+
+**CoqIDE**
+
+- Do not rely on the user settings but on the actual window size
```

Comment:
I think this describes more the solution than the problem. I would say something like:

```suggestion
- Fixed uneven dimensions of CoqIDE panels when window has been resized
```